Description: The dbus-appinfo test hangs in a build environment (with no X serveR)
Author: Iain Lane <laney@debian.org>
Forwarded: yes (desrt)

--- a/gio/tests/Makefile.am
+++ b/gio/tests/Makefile.am
@@ -385,7 +385,6 @@
 
 test_programs += \
 	actions					\
-	dbus-appinfo				\
 	gapplication				\
 	gdbus-auth				\
 	gdbus-bz627724				\
